doc/md_docs.go

```diff
@@ -72,6 +72,17 @@ func GenMarkdownCustom(cmd *cobra.Command, w io.Writer, linkHandler func(string)
 		buf.WriteString(fmt.Sprintf("```\n%s\n```\n\n", cmd.UseLine()))
 	}
 
+	if len(cmd.Aliases) > 0 {
+		buf.WriteString("Aliases: ")
+		for i, alias := range cmd.Aliases {
+			if i > 0 {
+				buf.WriteString(", ")
+			}
+			buf.WriteString(fmt.Sprintf("`%s`", alias))
```

Comment:
Here I'm surprised by this syntax, that was already used in the file I concede

I feel like this would be more natural.

> [!NOTE]
> This is more a note to myself to look deeper than something I expect you to change

This could have been used

```suggestion
			fmt.Fprintf(buf, "`%s`", alias)
```

@commit111 please look at this that seems a better option 

https://github.com/spf13/cobra/pull/2271#discussion_r2060904663